src/apps/shared/commands/patches/m2_6879_create_deleted_respondents.py

```diff
@@ -0,0 +1,153 @@
+import uuid
+
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import Query
+
+from apps.answers.db.schemas import AnswerSchema
+from apps.workspaces.crud.user_applet_access import UserAppletAccessCRUD
+from apps.workspaces.db.schemas import UserAppletAccessSchema
+from apps.workspaces.domain.constants import Role
+from apps.workspaces.service.workspace import WorkspaceService
+from infrastructure.database import session_manager
+
+LOCAL_DB_PATCH_SQL = """
+        insert into user_applet_accesses (
+            id, 
+            is_deleted, 
+            "role", 
+            user_id, 
+            applet_id, 
+            owner_id, 
+            invitor_id, 
+            meta,
+            created_at,
+            updated_at,
+            migrated_date 
+        )
+        select 
+            gen_random_uuid(),
+            true,
+            'respondent',
+            mr.user_id,
+            mr.applet_id,
+            mr.owner_id,
+            mr.invitor_id,
+            '{"secretUserId": "#deleted#"}'::jsonb,
+            now() AT TIME ZONE 'UTC',
+            now() AT TIME ZONE 'UTC',
+            mr.migrated_date
+        from (
+            select distinct 
+                a.respondent_id as user_id, 
+                a.applet_id as applet_id, 
+                aa.user_id as owner_id, 
+                aa.user_id as invitor_id,
+                a.migrated_date as migrated_date
+            from answers a 
+            join user_applet_accesses aa 
+            on aa.applet_id = a.applet_id 
+            and aa."role" = 'owner'
+            where 
+                (a.respondent_id, a.applet_id)  not in (
+                    select user_id, applet_id  
+                    from user_applet_accesses uaa 
+                    where 
+                        uaa."role" = 'respondent'
+                )
+        ) as mr
+"""
+
+
+async def get_answers(session: AsyncSession):
+    query: Query = select(AnswerSchema)
```

Comment:
Please just select `distinct applet_id, respondent_id`. You don't need full table here